pages/common/virsh.md

```diff
@@ -0,0 +1,37 @@
+# virsh
+
+> Manage virsh guest domains.
+> More information: >https://libvirt.org/virshcmdref.html/>.
+> NOTE: The GuestID can be the id, name or UUID of the guest.
+
+- Connect to a hypervisor session:
+
+`virsh connect {{qemu://system}}`
+
+- List inactive and active domains:
+
+`virsh list --all`
+
+- Dump guest configuration file:
+
+`virsh dumpxl {{GuestID}} > {{guest.xml}}`
```

Comment:
is there a reason why `GuestID` is using capital letters?

Reply:
No, it's just a placeholder

Reply:
Might be good to change it to `guest_id` where possible to match the [token syntax guidelines](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/style-guide.md#token-syntax). 👍

Reply:
@pxgamer woops missed that one